website/docs/admin_colorspace.md

```diff
@@ -0,0 +1,193 @@
+
+![admin_colorspace_settings_global](assets/settings/admin_colorspace_settings_host_config_1.png)
+
+Example of task level file structure.
+
+![admin_colorspace_settings_global](assets/settings/admin_colorspace_settings_host_config_3.png)
+
+Example of shot level file structure.
+
+![admin_colorspace_settings_global](assets/settings/admin_colorspace_settings_host_config_4.png)
+
+Example of shots sequence level file structure.
+
+![admin_colorspace_settings_global](assets/settings/admin_colorspace_settings_host_config_5.png)
+
+Example of project level file structure.
+
+### OCIO config versioning
+![admin_colorspace_settings_global](assets/settings/admin_colorspace_settings_host_config_versioning_2.png)
+
+During production, it may be necessary to modify the OCIO config files. We recommend using a versioning system to keep track of changes. The path to the config file version used during publishing will be stored in the representation document. This allows you to track which version of the OCIO config file was used during publishing.
+
+![admin_colorspace_settings_global](assets/settings/admin_colorspace_settings_host_config_versioning_1.png)
+
+:::warning
+
+Please note that since representation data might be pointing at older versions, it's essential to maintain the same file path to avoid any confusion. Therefore, make sure that the path in the representation document points to the folder that contains all versions of the OCIO config file.
+
+:::
+
+#### OCIO config description
+![admin_colorspace_settings_host_config_versioning](assets/settings/admin_colorspace_settings_host_config_versioning_3.png)
+
+We recommend using the `description` field to store information about the changes made to the OCIO config file. The file is in YAML format, and it's possible to use multiple lines in the `description` field only if `|` is used at the end of the line. Additionally, we add the `origin` and `changes` fields to the file. This allows you to track the origin of the OCIO config file and the changes made to it.
+
+### File rules override
+![admin_colorspace_settings_host_filerules](assets/settings/admin_colorspace_settings_host_filerules_1.png)
+
+If global File rules are disabled, the host-level File rules are ignored.
+
+## Upgrading from previous versions
+Upgrading may not always be a straightforward process, and we understand that its complexity can be frustrating. We strive to keep the upgrade process as simple as possible and to avoid any breaking changes. However, sometimes it's not possible, and we may need to introduce some breaking changes. This section aims to help you understand what needs to be done after the upgrade.
+
+### 1. Example of usage - [DCCs](admin_colorspace#used-acronyms) are set to native linear rec709 colorspace
+
+#### Actual configuration:
+A studio is having on-going production with project configured imageio at op 3.15. All hosts are set to `linear rec709` space with following combination of settings:
```

Comment:
Note that "is having" isn't typical english grammar. I'd replace all of them with `has` or `have` or `was` depending on the case. I'm not commenting them individually because there's quite a few of them.